themes/jquery/css/base.css

```diff
@@ -1970,7 +1970,7 @@ footer .books li a cite {
 
 .entry-meta,
 .entry-posted {
-	color: #999;
+	color: #666;
```

Comment:
These are used on multiple pages - `entry-meta` on some API metadata and `entry-posted` on metadata for blog posts, but both seem fine to me to change 👍🏻.